### ISSUE TITLE
Issue from QA: MCEDT and HCV issues

### ISSUE THREAD

**Issue by kateyang1998** [ccosp/open-o#111]
The original error from Magenta QA: NoClassDefFound error - this error was caused by the incompatibility of the ebs-client version. We modified the ebs-client source code and implemented the core classes into the OSCAR source code to solve the error.

The error "org.apache.cxf.binding.soap.SoapFault: An error was discovered processing the <wsse:Security> header" caused due to algorithm RSA1.5 being unsupported by the WSS4J 2.2.0 because of its security risk. This caused the header parsing error. We had to manually insert a line of code into the EDTClientBuilder.java to enable the algorithm RSA1.5.

The download function works after bypassing the security header validation - customized the WSS4J interceptor to catch the error.

**Comment by kateyang1998** [comment]
= Update =

All the errors were solved. 

The last error we encountered - "org.apache.ws.security.WSSecurityException: The signature or decryption was invalid", which caused when downloading the file was solved after adding the below properties into EdtClientBuilder.java in the newWSSInInterceptorConfiguration function:
- inProps.put(WSHandlerConstants.STORE_BYTES_IN_ATTACHMENT, "false");
- inProps.put(WSHandlerConstants.EXPAND_XOP_INCLUDE, "false");
After setting the STORE_BYTES_IN_ATTACHMENT and EXPAND_XOP_INCLUDE to false, the binary data is forced to be included inline in the SOAP message (STORE_BYTES_IN_ATTACHMENT, "false"), ensuring no issues arise from processing attachments. By setting EXPAND_XOP_INCLUDE to false, we avoid looking for XOP references in the response (explained by Deval).

Now, we're working on the MCEDT and HCV tests. There are 7 test cases in MCEDT tests encountering errors due to failing to pass the id with string type (the parameter type was set as <BigInteger>), and there are 18 test cases in HCV tests skipped due to those likely requests the online environment to run the test cases. We need to modify those test cases if we don't want to skip or fail any of them, or we have to leave them being skipped for now (TBD).
